Doc/library/collections.rst

```diff
@@ -33,10 +33,10 @@ Python's general purpose built-in containers, :class:`dict`, :class:`list`,
 :class:`UserString`     wrapper around string objects for easier string subclassing
 =====================   ====================================================================
 
-.. versionchanged:: 3.3
+.. deprecated-removed:: 3.3 3.9
     Moved :ref:`collections-abstract-base-classes` to the :mod:`collections.abc` module.
     For backwards compatibility, they continue to be visible in this module through
-    Python 3.7.  Subsequently, they will be removed entirely.
+    Python 3.7.
```

Comment:
Should this be changed to 3.8?

```suggestion
    Python 3.8.
```

Reply:
likely yes. I was unsure what this was trying to convey.

Reply:
My interpretation was that the import was still visible through `__getattr__` on module for 3.7 indicating 3.7 as last release to do so. But since 3.8 will continue to have that I suggested updating it to 3.8. I am not a native speaker though so this could be reverted if incorrect.